timegrinder/timeGrinder.go

```diff
@@ -186,15 +186,14 @@ func New(c Config) (tg *TimeGrinder, err error) {
 }
 
 func (tg *TimeGrinder) SetFormatOverride(v string) (err error) {
-	if v == `` {
```

Comment:
Didn't we just commit this change a week or two ago in the opposite direction?

Reply:
this matches the current state of master, which is what got the fix.  I don't think we brought the fix into dev, there was a little optimization on dev that I also brought in here.